### PR TITLE
test: sp db pkg adds unit test part II

### PR DESCRIPTION
**Pull request by sysvm**
### Description

sp db pkg adds unit test part II

### Rationale

Ensure SP code quality.

### Example

N/A

### Changes

Notable changes: 
* N/A

### CODE REVIEW

File: store/sqldb/traffic.go [hunk 1]
```diff
@@ -57,10 +57,12 @@ func (s *SpDBImpl) CheckQuotaAndAddReadRecord(record *corespdb.ReadRecord, quota
 
 	bucketTraffic, err := s.GetBucketTraffic(record.BucketID)
 	if err != nil && !errors.Is(err, gorm.ErrRecordNotFound) {
+		fmt.Println(1)
```
Comment (will-2012) [inline store/sqldb/traffic.go:60]: Remove it.

File: store/sqldb/traffic.go [hunk 2]
```diff
@@ -57,10 +57,12 @@ func (s *SpDBImpl) CheckQuotaAndAddReadRecord(record *corespdb.ReadRecord, quota
...
 		return err
 	}
 	// bucket traffic table should be initialized before checking meta
 	if bucketTraffic == nil {
+		fmt.Println(2)
```
Comment (will-2012) [inline store/sqldb/traffic.go:65]: Ditto.

**Review by flywukong — APPROVED** [review]
L

File: store/sqldb/traffic.go (hunk 1 above)
Comment (sysvm) [reply store/sqldb/traffic.go:60]: deleted

File: store/sqldb/traffic.go (hunk 2 above)
Comment (sysvm) [reply store/sqldb/traffic.go:65]: deleted

**Pull request closed without merging**